portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -916,6 +916,8 @@ definition {
 			value1 = ${selectOption});
 
 		if ((${fieldName} == "Choose a Data Provider") || (${fieldName} == "Output Parameter")) {
+			WaitForElementPresent(locator1 = "FormFields#SELECT_FIELD_DATA_PROVIDER_LIST");
+
```

Comment:
@davisaints do we need this?

Reply:
@davisaints also on CI?

Reply:
No, only locally. For now I will remove this, if there is a problem with this I will quickly correct it